pangeo-notebook/environment.yml

```diff
@@ -31,9 +31,11 @@ dependencies:
  - fsspec
  - gcm_filters
  - gcsfs
+ - gdal
  - gh
  - gh-scoped-creds
- - geocube
+ # temporarily drop for numpy>=2 (can use xvec instead)
+ #- geocube
```

Comment:
Interesting, so geocube pinned to `numpy<2` here - https://github.com/corteva/geocube/pull/175, but the incompatibility should have been fixed in rasterio 1.4 at https://github.com/rasterio/rasterio/pull/3109 if I'm reading correctly. Could open an issue on `geocube` to check, but ok to temporarily remove for now.

Reply:
> but the incompatibility should have been fixed in rasterio 1.4 at https://github.com/rasterio/rasterio/pull/3109 if I'm reading correctly. Could open an issue on geocube to check, but ok to temporarily remove for now. 

That would be great! I took the lazy approach just to see if this was the last remaining hold-out :)

Reply:
Opened a PR to remove the numpy<2 pin at https://github.com/corteva/geocube/pull/179 :crossed_fingers:

Reply:
Geocube 0.7.0 is out - https://github.com/conda-forge/geocube-feedstock/pull/41. I'll update the environment.yml file and re-lock